src/Aspire.Dashboard/Components/ResourcesGridColumns/LogLevelColumnDisplay.razor.cs

```diff
@@ -0,0 +1,34 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using Aspire.Dashboard.Otlp.Model;
+using Aspire.Dashboard.Resources;
+using Microsoft.JSInterop;
+
+namespace Aspire.Dashboard.Components;
+
+public partial class LogLevelColumnDisplay
+{
+    private bool TryGetErrorInformation([MaybeNullWhen(false)] out string errorInfo)
+    {
+        if (LogEntry.Properties.GetValue("ex.Type") is { } exceptionType)
```

Comment:
Non-standard property names. Could fallback to them, e.g. look for exception.type first and then ex.Type. Need to figure out why ex.Type is here at all first.

Reply:
@Erarndt is there a reason these property names are used? They come from here: https://github.com/dotnet/aspire/blob/0282715bfee52b3ed982228a67811b62ec8cb6b3/src/Components/Aspire.RabbitMQ.Client/RabbitMQEventSourceLogForwarder.cs#L156

Reply:
Falling back to them

Reply:
Create an issue for this.